javascript-light-bulb/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>JavaScript Light Bulb</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+<div>
+  <span></span>
+</div>
```

Comment:
While this works fine because there are no other elements, you want to get into the habit of using class names for your elements.